Traffic Routing Step Reference done (#5416)
harness/developer-hub: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/canary-deployment-step.md

If the **Skip Dry Run** option is selected, Harness will not use the `--dry-run` flag.

## Traffic Routing Configuration

> Configure traffic routing for canary deployment pipelines. For more information, see the [Traffic Routing Step Reference page](/docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/traffic-shifting-step.md).

Reviewer: ```suggestion
For information on how to configure traffic routing for Canary deployments, see the [Traffic Routing Step Reference page](/docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-k8s-ref/traffic-shifting-step.md).
```
